dropwizard-metrics-influxdb/src/main/java/com/izettle/metrics/dw/InfluxDbReporterFactory.java

```diff
@@ -72,6 +73,16 @@
  *         <td><i>None</i></td>
  *         <td>A boolean to signal whether to group gauges when reporting to InfluxDb.</td>
  *     </tr>
+ *     <tr>
+ *         <td>measurementMappings</td>
+ *         <td><i>None</i></td>
+ *         <td>A map for measurement mappings to be added, overridden or removed from the defaultMeasurementMappings.</td>
+ *     </tr>
+ *     <tr>
+ *         <td>defaultMeasurementMappings</td>
+ *         <td><i>None</i></td>
```

Comment:
I think it's important to document the defaults, maybe after this `<table>` we can explain them (still in the javadoc).